app/Components/Sudoku Board/SudokuBoard.tsx

```diff
@@ -438,13 +440,69 @@ const SudokuBoard = (props: SudokuBoardProps) => {
     const inputValue = event.nativeEvent.key;
     if (/^[1-9]$/.test(inputValue)) {
       updateCellEntry(parseInt(inputValue, 10));
-    }
-    if (
+    } else if (
       inputValue == "Delete" ||
       inputValue == "Backspace" ||
-      inputValue == "0"
-    )
+      inputValue == "0" ||
+      inputValue == "e" ||
+      inputValue == "E" // e and E are for erase
+    ) {
       eraseSelected();
+    } else if (inputValue == "u" || inputValue == "U") {
+      undo();
+    } else if (inputValue == "p" || inputValue == "P") {
+      saveGame(sudokuBoard);
+      navigation.goBack();
+    } else if (
+      inputValue == "t" ||
+      inputValue == "T" ||
```

Comment:
I don't know if I agree with "t" as a hotkey for toggle notes. Maybe "n" and "N" can be good enough and at some point we can allow configuration of the hotkeys in the settings?

Reply:
My reason for this wasn't only cause t for toggle. It was also that if you are navigating with WASD with one hand it would be really nice to be able to reach a toggle shortcut with that same hand while keeping other hand on numpad.
Thoughts?

Reply:
ok, I'm fine with keeping t as a hotkey.